framework/cmm-interface.md

```diff
@@ -106,17 +106,19 @@ The encryption materials returned MUST include the following:
 - [Encryption Context](structures.md#encryption-context)
   - The CMM MAY modify the encryption context.
 
-If the algorithm suite contains a [signing algorithm](algorithm-suites.md#signature-algorithm):
-
-- The CMM MUST include a [signing key](structures.md#signing-key).
```

Comment:
This is referring to the returned algorithm suite, which is required.

Reply:
I see, I'll clarify that.